app/models/Classroom.coffee

```diff
@@ -32,3 +32,59 @@ module.exports = class Classroom extends CocoModel
     }
     _.extend options, opts
     @fetch(options)
+    
+  getLevels: (options={}) ->
+    # options: courseID, withoutLadderLevels
+    Levels = require 'collections/Levels'
+    courses = @get('courses')
+    return new Levels() unless courses
+    levels = []
+    for course in courses
+      if options.courseID and options.courseID isnt course._id
+        continue
+      levels.push(course.levels)
+    levels = new Levels(_.flatten(levels))
```

Comment:
minor thing: I find it more confusing when one variable switches between types partway through a method